src/main/java/org/codehaus/mojo/versions/DisplayDependencyUpdatesMojo.java

```diff
@@ -198,20 +192,19 @@
     private static Set<Dependency> removeDependencyManagment( Set<Dependency> dependencies, Set<Dependency> dependencyManagement )
     {
         Set<Dependency> result = new TreeSet<>( new DependencyComparator() );
-        for ( Iterator<Dependency> i = dependencies.iterator(); i.hasNext(); )
+        for ( Dependency c : dependencies )
```

Comment:
when touching this and while at it, maybe make this variable more descriptive?
(Don't merge this suggestion, I will not createa them for the other occurences).

```suggestion
        for ( Dependency dependency : dependencies )
```